tests/test_modeling_common.py

```diff
@@ -1539,6 +1552,44 @@ def check_same_values(layer_1, layer_2):
             # self.assertTrue(model.transformer.wte.weight.shape, model.lm_head.weight.shape)
             # self.assertTrue(check_same_values(model.transformer.wte, model.lm_head))
 
+    def test_tied_model_weights_key_ignore(self):
+        config, inputs_dict = self.model_tester.prepare_config_and_inputs_for_common()
+        for model_class in self.all_model_classes:
+            model_tied = model_class(config)
+            with tempfile.TemporaryDirectory() as d:
+                model_tied.save_pretrained(d)
+
+                # We are nuking ALL weights on file, so every parameter should
+                # yell on load. We're going to detect if we yell too much, or too little.
+                with open(os.path.join(d, "pytorch_model.bin"), "wb") as f:
+                    torch.save({}, f)
```

Comment:
A comment here would be nice, as this is unusual.

Reply:
I gave you the wrong link (on an old commit, the comment is actually there).

Is it readable ?

Reply:
Oh, yes! Sorry,  I actually didn't understand that `We are nuking ALL weights on file, so every parameter should ...` which is quite clear already.